### PR TITLE
Login form

### PR DESCRIPTION
**Pull request by Nilmanduil**

### CODE REVIEW

File: src/pages/login/login.ts [hunk 1]
```diff
@@ -15,28 +17,38 @@ import { SignUpPage } from '../sign-up/sign-up';
...
+    if (this.loginForm.valid) {
+      this.auth.login(this.loginForm.value.phone, this.loginForm.value.password)
+        .then((token) => {
+          console.log('RESULT', token);
+        })
+        .catch((httpErrorResponse) => {
+          console.log('ERROR', httpErrorResponse.error.message);
+        });
+
+      this.viewCtrl.dismiss().then();
```
Comment (nicolasvergoz) [inline src/pages/login/login.ts]: Miss placed line of code.
This block allow any valid form to login.

File: src/pages/login/login.ts [hunk 2]
```diff
@@ -42,12 +42,12 @@ export class LoginPage {
       this.auth.login(this.loginForm.value.phone, this.loginForm.value.password)
         .then((token) => {
           console.log('RESULT', token);
+
```
Comment (nicolasvergoz) [inline src/pages/login/login.ts]: Unnecessary empty line

File: src/pages/login/login.ts [hunk 3]
```diff
@@ -42,12 +42,12 @@ export class LoginPage {
...
+          this.viewCtrl.dismiss().then();
         })
         .catch((httpErrorResponse) => {
           console.log('ERROR', httpErrorResponse.error.message);
         });
-
```
Comment (nicolasvergoz) [inline src/pages/login/login.ts]: Same here

**Comment by Nilmanduil** [comment]
Wake up @nicolasvergoz

**Pull request merged**